### ISSUE TITLE
[Feature] - Support One-click pipeline submissions for given backend

### ISSUE THREAD

**Issue by alefesta** [k3ai/k3ai#10]
We should have a command like `k3ai run` that allows a user to submit a code artifact to certain backends.
This would allow the user to test immediately with the AI platform and focus on the result. The command should look like:

```bash
k3ai run --source [REMOTE GIT URL] --backend [kfp/mlflow/airflow] --target [CLUSTER NAME] --type [py/yaml/zip]
```

Ideally, the minimal backend to support are:
 - [X] Kubeflow pipelines (argo)
 - [X] MLFlow
 - [ ] Apache Airflow

For the type (in order to understand what to do):
- py (python script)
- yaml
- zip (dsl in kfp)

**Comment by alefesta** [comment]
This is completed once we got:
- [ ] #14 
- [ ] #15 
- [x] MLFlow one click

**Comment by burntcarrot** [comment]
On a side note, can we add the Dockerfile for `k3ai-executor` to the repo? Code execution relies on the executor image, so it makes sense to make it available.

**Comment by alefesta** [comment]
Yep, I'm going to open a PR on this since I'm re-writing the executor image to use golang instead of a shell script to manage the backends. Should be open by tomorrow..
